Assets/HoloToolkit/Sharing/Scripts/Utilities/SharingServiceStatus.cs

```diff
@@ -0,0 +1,61 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEngine.UI;
+
+namespace HoloToolkit.Sharing
+{
+    public class SharingServiceStatus : MonoBehaviour
+    {
+        enum Status
+        {
+            Disconnected,
+            Connecting,
+            Connected,
+        }
+
+
+        [Tooltip("Reference to Text UI control where the Status info should be displayed.")]
+        public Text Text;
```

Comment:
Recommend using TextMesh instead. It's cheaper than Text which has canvas renderer stuff. I learned about this very recently and it makes atleast 5 fps difference :)